graphics/ipe/Portfile

```diff
@@ -22,9 +22,10 @@ master_sites            https://dl.bintray.com/otfried/generic/ipe/${branch}
 distname                ${name}-${version}-src
 worksrcdir              ${name}-${version}/src
 
-checksums           sha1    58410e29c04e7a7c3978bd9afe6d25a7924bd30d \
-                    rmd160  6d68ad7831923d04ba07086f3dde2be9830d756a \
-                    sha256  d66d3f2619e3e6ff617f42c2e3695c3db6e2a64adcc3d7613214e5fd14c49f22
+checksums           sha1    c8b82b6dc8bc286ad8f610f01e954e2bdd6ac65d \
```

Comment:
Please remove the sha1 checksum. These days, we favour rmd160, sha256 and size only.